content/news/030/index.md

```diff
@@ -85,6 +85,23 @@ Twitch][rust-gamedev-twitch].
 
 ## Game Updates
 
+### Country Slice
+
+![country_slice](country_slice_7_delete_brush.gif)
+
+[Country Slice][country-slice-github] is
+[@anastasiaopara][country-slice-twitter]'s hobby project, where users can draw a
+small procedurally assembled scene.
+
+Newest addition is an erase brush and an ability to continue existing walls.
+
+Discussions: procedural in Houdini vs Rust & OpenGL 👉
```

Comment:
The "Discussions: ..." format is usually used for public discussions so I'd rephrase that as something like "[@anastasiaopara][country-slice-twitter] also shared [a Twitter thread][country-slice-discussion] about procedural in Houdini vs Rust & OpenGL."

Reply:
Sounds good 👍